tutor/src/models/app/pulse-insights.js

```diff
@@ -14,6 +15,9 @@ class PI {
 
 export default {
   boot() {
+    if (!Flags.pulse_insights) {
+      return Promise.resolve({});
+    }
     window.pi = new PI();
```

Comment:
Hmm, were does PI come from?

Reply:
Seems to be the class above

Reply:
yeah, it's from the class, this is what CNX does I pretty much just copied from them